app/views/layouts/_simple_autocomplete.html.haml

```diff
@@ -0,0 +1,7 @@
+= f.input "#{association.to_s}_id", as: :hidden
+= f.input association, as: :hidden
```

Comment:
adding the associated field here so that if we get any errors on the association we can pull them up to the autocomplete field